styles.css

```diff
@@ -1199,6 +1345,21 @@ textarea:focus {
   textarea {
     width: 340px;
   }
+
+  /* .modal-button {
+    margin-top: -86px;
+    margin-left: 612px;
+  } */
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: